### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by ok-skalko**

### CODE REVIEW

File: lesson_1/home_work_1.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      puts 'No such file'
+    end
+  end
+end
+
+HomeWork.timezone
+HomeWork.color_text
+HomeWork.common_elements
+HomeWork.memory_size
+HomeWork.line_number
```
Comment (MrCartoon) [inline lesson_1/home_work_1.rb]: Please move these lines into separate files, cause when someone tries to run this code all the tasks will run at once.
Also, it's good to replace ARGV with the method's attributes & provide it while calling a method.

**Review by MrCartoon — CHANGES_REQUESTED** [review]
Try to do 6-8 tasks, they are good for training in the common use of ruby.

File: lesson_1/home_work_1.rb [hunk 2]
```diff
@@ -5,53 +5,78 @@ class HomeWork
...
     rescue Errno::ENOENT
       puts 'No such file'
     end
+
+    def random_element(arr)
+      puts arr.split('').sample
+    end
+
+    def sorted_array(arr)
+      puts arr.split('').sort.reverse
```
Comment (MrCartoon) [inline lesson_1/home_work_1.rb]: Here will be only sorted characters, for example, if I run `ruby sorted_array.rb a,b,c`, I will get `cba,,`.
You already did this in `common_elements`, so please fix it.

File: lesson_1/home_work_1.rb [hunk 3]
```diff
@@ -5,53 +5,78 @@ class HomeWork
...
           puts i + 1 if match_line
           break if match_line
         end
       end
     rescue Errno::ENOENT
       puts 'No such file'
     end
+
+    def random_element(arr)
+      puts arr.split('').sample
```
Comment (MrCartoon) [inline lesson_1/home_work_1.rb]: Same as `sorted_array`.

File: lesson_1/home_work_1.rb [hunk 4]
```diff
@@ -5,53 +5,78 @@ class HomeWork
...
+      def numbers(num1, num2, method)
+        a = num1.to_i
+        b = num2.to_i
+        case method
+        when 'sum'
+          puts a + b
+        when 'difference'
+          puts a - b
+        when 'division'
+          puts a / b
```
Comment (MrCartoon) [inline lesson_1/home_work_1.rb:72]: I want to divide 1 by 2 and get 0.5, but I'm getting 0. Find out how to fix this.

**Review by MrCartoon — CHANGES_REQUESTED** [review]

File: lesson_1/home_work_1.rb (hunk 2 above)
Comment (ok-skalko) [reply lesson_1/home_work_1.rb]: fixed

File: lesson_1/home_work_1.rb (hunk 3 above)
Comment (ok-skalko) [reply lesson_1/home_work_1.rb]: fixed

File: lesson_1/home_work_1.rb (hunk 4 above)
Comment (ok-skalko) [reply lesson_1/home_work_1.rb:72]: replace to_f instead to_i

File: lesson_1/home_work_1.rb (hunk 1 above)
Comment (ok-skalko) [reply lesson_1/home_work_1.rb]: done

**Review by MrCartoon — APPROVED** [review]
Great job!

**Comment by MrCartoon** [comment]
You can merge this pull request, cause right now I don't have required permissions.

**Pull request merged**